meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -1,183 +1,213 @@
 inherit image_types
 
-# Images inheriting this class MUST define:
+#
+# Create a raw image that can by written onto a storage device using dd/etcher.
+#
 # RESIN_IMAGE_BOOTLOADER        - bootloader
-# RESIN_BOOT_PARTITION_FILES    - list of items describing files to be deployed on boot partition
+# RESIN_BOOT_PARTITION_FILES    - list of items describing files to be deployed
+#                               - on boot partition
```

Comment:
If it is not and it's because you made the lines shorter, then was it supposed to be starting with a "-" ?